### PR TITLE
Change text on pricing and about us panels

### PR DESCRIPTION
**Pull request by qururoland**
Please review the revised text on the About Us and Pricing panels. There is a new panel to add so this needs merging first.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -493,39 +493,32 @@ <h1>
                     </h1>
                   </div>
 
-                  <p>
-                    QIS is the most economical option for your business with the easiest implementation. Manage over 5
-                    million images with no additional software. There are no licenses or usage costs for volumes beyond
-                    the traditional low traffic solutions.
-                  </p>
+                  <p>QIS is the most economical option for your business withQIS is the most economical option, with the easiest implementation for businesses that need to serve images at scale.   Manage over 40 million images with no additional software. There are no licenses or usage costs for volumes beyond the traditional low traffic solutions.</p>
```
Comment (fozcode) [inline index.html]: > QIS is the most economical option for your business withQIS is the most economical option

That needs fixing

File: index.html [hunk 2]
```diff
@@ -493,39 +493,32 @@ <h1>
...
+                  <h2>Fully open source Standard Eddition - FREE</h2>
```
Comment (fozcode) [inline index.html]: > Eddition

Typo

File: index.html (hunk 1 above)
Comment (fozcode) [reply index.html]: 40 million seems a bit random.

> There are no licenses or usage costs for volumes beyond the traditional low traffic solutions

That doesn't make much sense to me - there is usage cost for hosting and maybe bandwidth, and I don't know what a "traditional low traffic solution" is.  But I won't argue if it's a marketing thing.

Comment (qururoland) [reply index.html]: Duplicate text should now be removed...

File: index.html (hunk 2 above)
Comment (qururoland) [reply index.html]: Now with one D

**Pull request merged**